src/php/ext/grpc/php_grpc.h

```diff
@@ -91,8 +92,13 @@ ZEND_END_MODULE_GLOBALS(grpc)
 #define GRPC_G(v) (grpc_globals.v)
 #endif
 
+ZEND_DECLARE_MODULE_GLOBALS(grpc)
```

Comment:
I'm not certain, but I feel like doing this in a ```.h``` files might have weird side effects.